### PR TITLE
bug: fix ctip parser handling JSON Payload content

### PR DESCRIPTION
**Pull request by wagner-intevation**
Handle Payload field with non-base64-encoded JSON content and numbered dictionaries

### CODE REVIEW

File: intelmq/etc/feeds.yaml [hunk 1]
```diff
@@ -723,21 +723,21 @@ providers:
...
-        
+
         PGP Public key:
         ```
         -----BEGIN PGP PUBLIC KEY BLOCK-----
         Version: SKS 1.1.6
         Comment: Hostname: pgp.mit.edu
-        
+
         mQINBFRl7D8BEADaRFoDa/+r27Gtqrdn8sZL4aSYTU4Q3gDr3TfigK8H26Un/Y79a/DUL1o0
```
Comment (waldbauer-certat) [inline intelmq/etc/feeds.yaml]: Intended removal of whitespaces?

File: intelmq/etc/feeds.yaml [hunk 2]
```diff
@@ -1887,10 +1887,10 @@ providers:
                       listen 443 ssl http2;
                       server_name [your host name];
                       client_max_body_size 50M;
-                      
+
                       ssl_certificate [path to your key];
                       ssl_certificate_key [path to your certificate];
-                      
+
                       location /[your private url] {
```
Comment (waldbauer-certat) [inline intelmq/etc/feeds.yaml]: Same here

File: intelmq/etc/feeds.yaml (hunk 1 above)
Comment (wagner-intevation) [reply intelmq/etc/feeds.yaml]: No, not intentional. Removing the whitespace does not change the output though. Currently the rendered output of this section is broken: https://intelmq.readthedocs.io/en/latest/user/feeds.html#greylist-with-pgp-signature-verification

**Review by waldbauer-certat — APPROVED** [review]
LGTM

**Pull request merged**